File: Android/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/network/core/ResourceType.java
package com.didichuxing.doraemonkit.kit.network.core;

public enum ResourceType {
    DOCUMENT("Document"),
    STYLESHEET("Stylesheet"),
    IMAGE("Image"),
    FONT("Font"),
    SCRIPT("Script"),
    XHR("XHR"),
    WEBSOCKET("WebSocket"),
    OTHER("Other");

    private final String mProtocolValue;

    private ResourceType(String protocolValue) {
      mProtocolValue = protocolValue;
    }

    public String getProtocolValue() {
      return mProtocolValue;
    }
  }

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/network/okhttp/core/ResourceType.kt
package com.didichuxing.doraemonkit.kit.network.okhttp.core

enum class ResourceType(val protocolValue: String) {
    DOCUMENT("Document"), STYLESHEET("Stylesheet"), IMAGE("Image"), FONT("Font"), SCRIPT("Script"), XHR("XHR"), WEBSOCKET("WebSocket"), OTHER("Other");

}